resdk/analysis/prepare_geo.py

```diff
@@ -0,0 +1,160 @@
+"""Prepare GEO."""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import time
+import traceback
+
+from resdk.resources.utils import (
+    get_resource_collection, get_samples, get_resolwe
+)
+
+__all__ = ('prepare_geo_chipseq', 'prepare_geo_rnaseq', 'prepare_geo')
+
+
+def is_background(sample):
+    """Return ``True`` if given sample is background and ``False`` otherwise."""
+    background_relations = sample.resolwe.relation.filter(
+        type='compare',
+        label='background',
+        entity=sample.id,
+        position='background'
+    )
+    return len(background_relations) > 0
+
+
+def get_name_collection(collection_ids, res):
```

Comment:
This is really strange function. If input is as list of collection id's, you should also return list of collection names. If you anly support a list with single element, input should be a single collection object, not a list of them. First decide on this. 

Rename this function to `get_collection_name`, i think it explains the functionality better.